pkgs/applications/backup/unifi-protect-backup/default.nix

```diff
@@ -13,15 +15,20 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-VIA7/jWMuAWYle3tBDgKU5PJ9wkHChEjNns8lhYr1K8=";
   };
 
-  preBuild = ''
-    sed -i 's_click = "8.0.1"_click = "^8"_' pyproject.toml
-    sed -i 's_pyunifiprotect = .*_pyunifiprotect = "*"_' pyproject.toml
-    sed -i 's_aiorun = .*_aiorun = "*"_' pyproject.toml
-    sed -i '/pylint/d' pyproject.toml
-  '';
+  pythonRelaxDeps = [
+    "aiorun"
+    "aiosqlite"
+    "click"
+    "pyunifiprotect"
+  ];
```

Comment:
Uh, that can't possibly work. The hook doesn't handle anything but wheel metadata as it is.

Reply:
Wow, we edit the wheel before installPhase. Slightly disgusting.

Reply:
The output looked ok but it got me thinking about if I was using the hook in the wrong way in the past.